iscsidev/iscsi.go

```diff
@@ -167,6 +177,45 @@ func (dev *Device) StartInitator() error {
 	return nil
 }
 
+func (dev *Device) ReloadInitiator() error {
```

Comment:
Reloading doesn't need to relogin the target?

Reply:
This reload means initiating some fields for the struct rather than doing something for the iscsi

Reply:
Do I need to rename the function to avoid misunderstanding?

Reply:
Probably no need, but having a comment for this function will be good.

Reply:
Updated.